src/Microsoft.Framework.Runtime/DependencyManagement/ProjectReferenceDependencyProvider.cs

```diff
@@ -45,7 +45,7 @@ public LibraryDescription GetDescription(LibraryRange libraryRange, FrameworkNam
 
             // This never returns null
             var targetFrameworkInfo = project.GetTargetFramework(targetFramework);
-            var targetFrameworkDependencies = targetFrameworkInfo.Dependencies;
+            var targetFrameworkDependencies = new List<LibraryDependency>(targetFrameworkInfo.Dependencies);
```

Comment:
Instead of fixing it here. Change the thing fix it this in the project class itself.